app/src/main/java/com/example/mycalculator/mvp/CalculatorViewImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.example.mycalculator.mvp;
+
+import com.example.mycalculator.MainActivity;
+import com.example.mycalculator.databinding.ActivityMainBinding;
+
+public class CalculatorViewImpl extends ActivityView implements CalculatorContract.CalculatorViewContract {
```

Comment:
you can change the name of this class and call it CalculatorView

Reply:
Done. Thx!